frontend/src/metabase-lib/v1/queries/NativeQuery.ts

```diff
@@ -313,6 +311,18 @@ export default class NativeQuery extends AtomicQuery {
     return tagErrors.length === 0;
   }
 
+  private _allRequiredTagsHaveDefaults() {
+    const isEmpty = v => {
```

Comment:
Is there an existing function to detect an empty value (empty being anything false-y in an array or standalone) I could use here?